doc/conseil.sql

```diff
@@ -189,7 +189,31 @@ CREATE TABLE public.balance_updates (
     category character varying
 );
 
+CREATE TABLE public.delegates (
+    pkh character varying PRIMARY KEY,
+    block_id character varying NOT NULL,
+    balance numeric,
+    frozen_balance numeric,
+    staking_balance numeric,
+    delegated_balance numeric,
+    deactivated boolean NOT NULL,
+    grace_period integer NOT NULL,
+    block_level integer DEFAULT '-1'::integer NOT NULL
+);
 
+CREATE TABLE public.delegated_contracts (
```

Comment:
This would duplicate a lot of data from the `accounts` table. We are only supposed to store the IDs here for redundancy purposes.

Reply:
Ouch, fine, I would assume at least a reference to the delegate, too?

Reply:
@vishakh request for clarification:
which one is the relation field (FK) to join delegated contracts with their delegate?
I'm going for delegate.value

Reply:
Thanks to this change I found out another flaky test, duplicate from the one I fixed earlier! 🍪

Reply:
> Ouch, fine, I would assume at least a reference to the delegate, too?

Indeed.

Reply:
> @vishakh request for clarification:
> which one is the relation field (FK) to join delegated contracts with their delegate?
> I'm going for delegate.value

You mean between the `delegated_contracts` and `delegates` tables? `delegated_contracts.account_id` should join to `delegates.pkh`.